assets/js/components/Work/Tabs/About/CoreMetadata.jsx

```diff
@@ -1,20 +1,23 @@
 import React from "react";
 import PropTypes from "prop-types";
-import UITagNotYetSupported from "../../../UI/TagNotYetSupported";
 import UIInput from "../../../UI/Form/Input";
 import UIFormField from "../../../UI/Form/Field";
 import UIFormSelect from "../../../UI/Form/Select";
 import UIFormFieldArray from "../../../UI/Form/FieldArray";
 import UIFormFieldArrayDisplay from "../../../UI/Form/FieldArrayDisplay";
 import UICodedTermItem from "../../../UI/CodedTerm/Item";
 import { useCodeLists } from "@js/context/code-list-context";
+import { isEDTFValid } from "../../../../services/helpers";
 
 const WorkTabsAboutCoreMetadata = ({
   descriptiveMetadata,
   isEditing,
   published,
 }) => {
   const codeLists = useCodeLists();
+  const EDTFValidateFn = (value) => {
+    return isEDTFValid(value) || <span>Please enter raw EDTF date. </span>;
```

Comment:
Looks like we can add a `<a href="" />` element here.  Can we add the link to the EDTF Library of Congress site?